CWE/events/italy.txt

```diff
@@ -54,460 +99,232 @@ country_event = {
-  title = EVT_8007740_NAME
-  desc = EVT_8007740_DESC
-  picture = "nwo2_italian_communist_party"
-  fire_only_once = yes
-
-	trigger = {
-		tag = ITA
-		year = 1950 NOT = { year = 1980 }
-		OR = { has_country_flag  = very_radical_ICP has_country_flag  = radical_ICP }
-		OR = { government = democracy government = hms_government }
-		NOT = { 
-			OR = { 
-				ruling_party_ideology = socialist 
-				ruling_party_ideology = communist 
-				ruling_party_ideology = communist_social 
-			} 
-		}
-		has_global_flag = blocsenabled
-	}
-
-	mean_time_to_happen = { months = 24 }
-
-	option = {
-		name = EVT_8007740_A
-		any_pop = { militancy = 2.0 consciousness = 5 }
-		relation = { who = USA value = 100 } 
-		country_event = 8007742
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-	}
-	
-	option = {
-		name = "Let the ICP remain legal"
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
-	}
-	
-}
-
-#Radicalization of Italian Communist Party
-country_event = {
-  id = 8007741
-  title = EVT_8007741_NAME
-  desc = EVT_8007741_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes
-
-	option = {
-		name = EVT_8007741_A
-		any_pop = { militancy = 1.0 consciousness = 1 }
-		set_country_flag  = very_radical_ICP
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.02 value = communist }
-		}
-	}
-  
-}
-
-#Dispersion of Italian Communist Party
-country_event = {
-  id = 8007742
-  title = EVT_8007742_NAME
-  desc = EVT_8007742_DESC
-  picture = "nwo2_italian_communist_party"
-  is_triggered_only = yes
-
-	option = {
-		name = EVT_8007742_A
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = -0.10 value = communist }
-		}
-	}
 	
 }
 
-#Italian Communist Party at helm
+#The Special Laws
 country_event = {
-  id = 8007743
-  title = EVT_8007743_NAME
-  desc = EVT_8007743_DESC
-  picture = "nwo2_italian_communist_party"
+	id = 8007772
+	title = "The $YEAR$ Special Laws"
+	desc = EVTDESC8007772
+	picture = "italian_parliament"
   
 	fire_only_once = yes
-  
+
 	trigger = {
 		tag = ITA
-		has_global_flag = blocsenabled
-		ruling_party_ideology = communist
-		government = proletarian_dictatorship
-		election = no
-	}
-
-	option = {
-		name = EVT_8007743_A
-		
-		#Break alliances with non communists
-		any_country = {
-			limit = { NOT = { ruling_party_ideology = communist } }
-			leave_alliance = THIS
-			relation = { who = THIS value = -200 }
-		}
-		
-		#Be friends with fellow communists
-		any_country = {
-			limit = { ruling_party_ideology = communist }
-			relation = { who = THIS value = 200 }
-		}
+		year = 1948
+		NOT = { year = 1990 }
 		
-		#Citizens become more communist
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = communist }
-		}
+		government = democracy
+		is_vassal = no
+		war = no
 		
```

Comment:
There should be a check for a communist or socialist ruling party here since this event is to create a proletarian dictatorship

Reply:
I feared that with the vanilla elections system the ruling party in that period might not actually be communist or socialist and that would prevent the event from firing even if all the other prerequisites are met, if it is necessary I will try to tweak events boosts to secure a solid majority for when the event fires

Reply:
It’s better to make sure a socialist or communist party is in power for that event to fire.